### PR TITLE
Fix #959: update transport parameters documentation

### LINKED ISSUE #959: Outdated content in EGSnrc manual (PIRS-701)
Some content in the EGSnrc manual `PIRS-701` needs updating. For example, Dave Rogers noted the following outdated passage regarding the default photon cross section:

> In the current version of EGSnrc, the user has the option to use photon
> cross section data other than the **_default Storm & Israel data._** To do this,
>   the user must set the character variable photon xsections (part of the
>   COMIN/MEDIA common block) to the name of the cross section data to use.
>   Cross sections included with the EGSnrc distribution are: i
>   1) the Evaluated Photon Data Library[44] (set photon xsections= "epdl")
>   2) XCOM data from Berger & Hubbell[37] (set photon xsections= "xcom" –
>      **_also the default in the absence of any input)_**

The bold passages are incorrect, given that the default was changed in 2012:
```diff
$ cvs diff -r1.48 -r1.49 src/egsnrc.macros
Index: src/egsnrc.macros
===================================================================
RCS file: /home/cvsroot/HEN_HOUSE/src/egsnrc.macros,v
retrieving revision 1.48
retrieving revision 1.49
diff -r1.48 -r1.49
2c2
< " $Id: egsnrc.macros,v 1.48 2012/10/04 23:15:00 Exp $ "
---
> " $Id: egsnrc.macros,v 1.49 2012/10/05 00:19:51 Exp $ "
2252c2252
< REPLACE {$XDATA-DEFAULT} WITH {'si'}
---
> REPLACE {$XDATA-DEFAULT} WITH {'xcom'}
```

### PR DESCRIPTION
**Pull request by ftessier**
Update the documentation in manuals and in code for the current EGSnrc Monte Carlo parameters and their default options. There was a lot of outdated content scattered throughout. I checked in the code that the defaults correspond to those in the code.

I created many commits to facilitate review. Each one corresponds to one particular option default. The first large one contain the typos and tweaks without any impact on the documentation of defaults.

Note that `dosxyznrc` has a different default value for the Boundary crossing algorithm (`PRESTA-I`, which changes the skin depth as well). I think this should be changed to `EXACT`, with a note in the documentation to explain how simulations may become faster with `PRESTA-I`, not the other way around with a justification! The default options should produce the most accurate results possible.

The second commit in the chain has to do with an update of defaults _in the code_ (and documentation) for the `IAPRIM` parameter. It should be `1` instead of `0`. I will move this to its own pull request before merging, to keep the current pull request as documentation only, without functional changes.

While at it, I noticed and corrected a number of typos throughout the EGSnrc tree as well.

This fixes #959.

### CODE REVIEW

**Comment by rtownson** [comment]
Rebased on develop.

**Comment by rtownson** [comment]
I've compile the documentation on this branch and updated the website for the 2023 release. So currently, all of these changes are already live. If we make any changes, just let me know and I will update the website again.

**Review by rtownson — APPROVED** [review]

File: HEN_HOUSE/doc/src/pirs701-egsnrc/inputs/transport_parameters.tex [hunk 1]
```diff
@@ -236,27 +234,32 @@
                         to NRC, then use NRC pair production cross-sections
                         (in file $HEN_HOUSE/data/pair_nrc1.data).  Only
                         of interest at low energies, where the NRC cross-
-                        sections take into account the assymmetry in the
+                        sections take into account the asymmetry in the
                         positron-electron energy distribution.
                         [ pair_nrc ]
```
Comment (blakewalters) [inline HEN_HOUSE/doc/src/pirs701-egsnrc/inputs/transport_parameters.tex:239]: Because we all know what happens when you assume assymmetry.

**Review by blakewalters — APPROVED** [review]

**Review by mainegra — APPROVED** [review]

**Comment by rtownson** [comment]
Fixed a minor conflict in the contributors list.

**Pull request merged**